packages/eliom/eliom.11.0.0/opam

```diff
@@ -0,0 +1,63 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Advanced client/server Web and mobile framework"
+description: """
+Eliom is a framework for implementing Web sites and client/server Web and mobile applications.
+It uses advanced concepts to simplify the implementation of common behaviors (e.g. scoped sessions, continuation based Web programming ...).
+It uses advanced static typing features of OCaml to check many properties of the Web application at compile-time (html, page parameters ...).
+Eliom allows implementing the whole application as a single program that includes both the client and the server code. For example, you can implement event handlers (onclick ...) directly in OCaml, and you can call a server-side OCaml function from the client. Pages are generated either on the server or the client.
+These client-side features remain compatible with traditional Web programming (links, forms, URLs, bookmarks, sessions ...).
+It is possible to generate mobile applications for Android and iOS with the exact same code as your Web application.
+The client-side code is compiled to JS using Ocsigen Js_of_ocaml or to Wasm using Wasm_of_ocaml.
+"""
+maintainer: ["dev@ocsigen.org"]
+authors: ["dev@ocsigen.org"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://ocsigen.org/eliom/"
+bug-reports: "https://github.com/ocsigen/eliom/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind"
+  "ppx_deriving"
+  "ppxlib" {>= "0.15.0"}
+  "js_of_ocaml-compiler" {>= "3.6.0"}
+  "js_of_ocaml" {>= "3.6.0"}
+  "js_of_ocaml-lwt" {>= "3.6.0"}
+  "js_of_ocaml-ocamlbuild" {build}
+  "js_of_ocaml-ppx" {>= "3.6.0"}
+  "js_of_ocaml-ppx_deriving_json" {>= "3.6.0"}
+  "js_of_ocaml-tyxml" {>= "3.6.0"}
+  "lwt_log"
+  "lwt_ppx" {>= "1.2.3"}
+  "tyxml" {>= "4.6.0" & < "4.7.0"}
+  "ocsigenserver" {>= "6.0.0" & < "7.0.0"}
+  "ipaddr" {>= "2.1"}
+  "reactiveData" {>= "0.2.1"}
+  "base-bytes"
+  "ocsipersist" {>= "2.0" & < "3.0"}
+  "ppx_optcomp"
+  "odoc" {with-doc}
```

Comment:
Ah, I am mistaken about the API of xml-light, seems that the needed module type was actually introduced in 2.5, so I think we either need

```suggestion
  "odoc" {with-doc}
  "xml-light" {>= "2.5"}
```

here or else in `ocisgenserver` (from which the xml-light dependency is being being picked up transitively).

Reply:
Thank you! I will add these constraints